### DISCUSSION TITLE
Lost 2FA and recovery codes

### DISCUSSION THREAD

**Discussion by bitnerrd** [community/community#52576] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I lost my github 2FA and recovery codes and 2FA application. Now I am locked out. I have genrated two tickets but there is no reply from github. I want my account back.

**Comment by airtower-luna** [comment]
There might be other options if you have a PAT or SSH key, see the [2FA recovery documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). But the documentation also says if you don't have any of the recovery options, there's nothing you can do:

> If you cannot use any recovery methods, you have permanently lost access to your account.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @binrynerd2 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by marcinn** [comment]
> If you cannot use any recovery methods, you have permanently lost access to your account.

So why is GitHub forcing me to lose access to my account?

**Comment by bitnerrd** [comment]
This is completely beyond my understanding and what kind of law is it that
if a guy has run out of recovery methods he has lost it.
Is this an account or a living person who dies and cannot come back to life?
In the name of strict security, this is a very mean way to waste people's
hard work.
